assets/src/components/v2/simulation_screen_container.tsx

```diff
@@ -11,21 +11,27 @@ import {
 
 interface SimulationScreenLayoutProps {
   apiResponse: ApiResponse;
+  opts: {[key: string]: any}
 }
 
 const SimulationScreenLayout: ComponentType<SimulationScreenLayoutProps> = ({
   apiResponse,
+  opts,
 }) => {
   const responseMapper = useContext(ResponseMapperContext);
   const data = responseMapper(apiResponse);
   const { fullPage, flexZone } = data;
+  
+  // If "alternatePage" was provided as an option, we use the simulation version of screen normal
+  // Currently only applies to DUPs
+  const alternatePage = {...fullPage, type: "simulation_screen_normal"}   
 
   return (
     <div className="simulation-screen-centering-container">
       <div className="simulation-screen-scrolling-container">
         {apiResponse && (
           <div className="simulation__full-page">
-            <Widget data={fullPage ?? data} />
+            { opts.alternateView ? <Widget data={alternatePage} /> : <Widget data={fullPage ?? data} /> }
```

Comment:
It's not terribly difficult to follow now, but this might be a little more clear if the value of `data` is evaluated in an if/else block above the return.